actionpack/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+*   Change `ActionDispatch::PermissionsPolicy` to generate both the
+    `Permissions-Policy` header and the `Feature-Policy` header.
+
+    This also adds the `interest-cohort` directive to the list of supported directives,
```

Comment:
Where is this being added? I could not find any reference to this directive in the code.